test/iden3comm/domain/use_cases/fetch_and_save_claims_use_case_test.dart

```diff
@@ -96,6 +110,13 @@ void main() {
           .thenAnswer((realInvocation) => Future.value(claimEntity));
       when(getClaimRevocationStatusUseCase.execute(param: anyNamed('param')))
           .thenAnswer((realInvocation) => Future.value(revStatus));
+      when(getDidIdentifierUseCase.execute(param: anyNamed('param')))
+          .thenAnswer((realInvocation) => Future.value(CommonMocks.did));
+      when(getEnvUseCase.execute(param: anyNamed('param')))
+          .thenAnswer((realInvocation) => Future.value(CommonMocks.env));
+      when(checkProfileAndDidCurrentEnvUseCase.execute(
+              param: anyNamed('param')))
+          .thenAnswer((realInvocation) => Future.value(null));
```

Comment:
Missing the params capture check of those.

Reply:
done